environment.yml

```diff
@@ -1,15 +1,18 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-pgdb-1.0dev
+name: nf-core-pgdb-1.0.0
 channels:
   - conda-forge
   - bioconda
   - defaults
 dependencies:
-  - conda-forge::python=3.7.3
-  - conda-forge::markdown=3.1.1
-  - conda-forge::pymdown-extensions=6.0
-  - conda-forge::pygments=2.5.2
-  # TODO nf-core: Add required software dependencies here
-  - bioconda::fastqc=0.11.8
-  - bioconda::multiqc=1.7
+  - conda-forge::python=3.9.2
```

Comment:
Could be sorted by conda channel. and pypgatk probably belongs to the pipeline not nfcore, right?

Reply:
PR #20